Samurai/Tasks/TickerManager.cs

```diff
@@ -1,15 +1,13 @@
 using System;
-using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using AcidChicken.Samurai.Components;
```

Comment:
All of the local namespace reference instructions have to be included in the local namespace block.